app/mailers/match_digest_mailer.rb

```diff
@@ -0,0 +1,43 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/boston-cas/blob/production/LICENSE.md
+###
+
+class MatchDigestMailer < ApplicationMailer
+  def digest(contact)
+    @matches = [
+      ['Stalled', :stalled],
+      ['Recently Canceled', :canceled],
+      ['Recently Expired', :expired],
+      ['Expiring Soon', :expiring],
+      ['Active', :active],
+    ].each_with_object({}) do |group, hash|
+      hash[group[1]] = { title: group[0], matches: [] }
+    end
```

Comment:
Sure seems like this would be easier to read as:
```suggestion
@matches = { 
  stalled: {
    title: 'Stalled',
    matches: [],
  },
  canceled: {
    title: 'Recently Canceled',
    matches: [],
  },
  expired: {
    title: 'Recently Expired',
    matches: [],
  },
  expiring: {
    title: 'Expiring Soon',
    matches: [],
  },
  active: {
    title: 'Active',
    matches: [],
  },
}
```